### PR TITLE
Clean up footer

### PR DESCRIPTION
**Pull request by zachlatta**
New:

new

Old:

old

### CODE REVIEW

File: app/assets/stylesheets/pages.css.scss [hunk 1]
```diff
@@ -319,25 +319,51 @@ $include-html-global-classes: false;
...
+      margin-left: 5px;
     }
 
-    a {
-      color: scale-color($text-on-black, $lightness: 10%);
-      text-decoration: underline;
+    ul {
+      margin-left: 0;
+
+      li {
```
Comment (zachlatta) [inline app/assets/stylesheets/pages.css.scss]: We're going to have to redesign our general CSS layout if we want to impose this in our codebase. Thought on ignoring this for now, @MaxWofford?

File: app/assets/stylesheets/pages.css.scss [hunk 2]
```diff
@@ -319,25 +319,51 @@ $include-html-global-classes: false;
...
       font-size: rem-calc(18);
-      margin: 15px 0 15px;
+      font-weight: 600;
+      margin-left: 5px;
     }
 
-    a {
-      color: scale-color($text-on-black, $lightness: 10%);
-      text-decoration: underline;
+    ul {
```
Comment (zachlatta) [inline app/assets/stylesheets/pages.css.scss]: Same thing as above applies here. We'd need a much larger redesign to account for cases like this.

File: app/assets/stylesheets/pages.css.scss [hunk 3]
```diff
@@ -319,25 +319,47 @@ $include-html-global-classes: false;
...
+
+        a {
+          color: $hackedu-white;
+          display: block;
+          height: 100%;
+          width: 100%;
+        }
+      }
+
+      .link:hover {
```
Comment (zachlatta) [inline app/assets/stylesheets/pages.css.scss]: Same thing as above applies here. I think we'd need a much larger redesign to follow this guideline

**Comment by MaxWofford** [comment]
@zachlatta Why is the attributions page removed?

**Comment by zachlatta** [comment]
We no longer use the assets we were attributing on the attributions page.

**Comment by MaxWofford** [comment]
:+1:

**Pull request merged**